server.js

```diff
@@ -20,6 +23,35 @@ app.get('/ping', (req, res) => {
   res.send('This is the About page');
 })
 
+
+// Connection URI
+const uri = 'mongodb+srv://NAYANKUMARRAJ:nkr2580@my-first-cluster.hz1puza.mongodb.net/?retryWrites=true&w=majority';

```

Comment:
Consider moving the database connection logic to a separate module. This will improve the modularity of your code and make it easier to manage. [medium]